### PR TITLE
Make more e2e test pass

### PR DESCRIPTION
**Pull request by xavier-rbz**
Adapt to new representation of geo data (asn, country...), and fix
bugs induced by these changes in lua code. 3 more e2e tests pass.

Signed-off-by: Xavier <xavier@reblaze.com>

### CODE REVIEW

File: curiefense/curieproxy/lua/tagprofiler.lua [hunk 1]
```diff
@@ -26,6 +26,28 @@ function match_singles(request_map, list_entry)
...
+          return entry_match
+        end
+      end
+    end
+
+    -- pattern matching for ASN
+    if entry_key == 'asn' then
+      local value = request_map.geo.asn
+      if value then
+        entry_match = list_entries[request_map.geo.asn]
```
Comment (flaper87) [inline curiefense/curieproxy/lua/tagprofiler.lua:44]:
```suggestion
        entry_match = list_entries[value]
```

File: curiefense/curieproxy/lua/tagprofiler.lua [hunk 2]
```diff
@@ -26,6 +26,28 @@ function match_singles(request_map, list_entry)
       return entry_match
     end
 
+    -- pattern matching for country
+    if entry_key == 'country' then
+      local value = request_map.geo.country.iso
+      if value then
+        entry_match = list_entries[request_map.geo.country.iso]
```
Comment (flaper87) [inline curiefense/curieproxy/lua/tagprofiler.lua:33]:
```suggestion
        entry_match = list_entries[value]
```

**Review by flaper87 — CHANGES_REQUESTED** [review]
a couple of nits since luacheck is failing :) 

Otherwise, LGTM

File: curiefense/curieproxy/lua/tagprofiler.lua (hunk 2 above)
Comment (xavier-rbz) [reply curiefense/curieproxy/lua/tagprofiler.lua]: sure, done in the next commit

File: curiefense/curieproxy/lua/tagprofiler.lua (hunk 1 above)
Comment (xavier-rbz) [reply curiefense/curieproxy/lua/tagprofiler.lua]: sure, done in the next commit

**Review by flaper87 — APPROVED** [review]
:spaghetti: 

We won't get luacheck to pass in this PR

**Pull request merged**